docs/data-sources/cluster.md

```diff
@@ -80,7 +80,7 @@ data "tanzu-mission-control_cluster" "read_tkg_aws_cluster" {
 
 ### Optional
 
-- `attach_k8s_cluster` (Block List, Max: 1) (see [below for nested schema](#nestedblock--attach_k8s_cluster))
+- `attach_k8s_cluster` (Block List, Max: 2) (see [below for nested schema](#nestedblock--attach_k8s_cluster))
```

Comment:
why is this max two ?

Reply:
One element is description, one is the kube_config input (either of the 2 available inputs), so that user doesn't provide multiple inputs.

Reply:
this limit is for the entries in they list. 
kuebconfig and description together will be on entry.

Reply:
Done